sdk/python/tests/unit/permissions/auth/server/conftest.py

```diff
@@ -70,7 +149,11 @@ def _include_auth_config(file_path, auth_config: str):
     print(f"Updated auth section at {file_path}")
 
 
-def _default_store(temp_dir, auth_config: str):
+def _default_store(
+    temp_dir,
+    auth_config: str,
+    permissions: Optional[Union[Permission, list[Permission]]],
```

Comment:
I think it's never a `Permission` type, according to the value of the above `params`.
Also, with a `Permission` type it would probably fail because of some code like `len(permissions)` that would fail

Reply:
Fixed